src/utils/message_builder.rs

```diff
@@ -723,6 +727,190 @@ impl fmt::Display for MessageBuilder {
     }
 }
 
+
+/// Content modifier enum
+///
+/// Provides an enum of formatting modifiers for a string, for combination with
+/// string types and Content types.
+///
+/// # Examples
+///
+/// Create a new Content type which describes a bold-italic "text":
+///
+/// ```rust
+/// use serenity::utils::{Bold, Italic, Content};
+/// let content: Content = Bold + Italic + "text";
+/// ```
+pub enum ContentModifier {
+  Italic,
+  Bold,
+  Strikethrough,
+  Code,
+  Underline
+}
+
+/// Describes formatting on string content
+#[derive(Default, Clone)]
+pub struct Content {
+  italic: bool,
+  bold: bool,
+  strikethrough: bool,
+  pub inner: String,
+  code: bool,
+  underline: bool
```

Comment:
Although there's never a reason to mutate or read these fields directly, we should expose them to users. Unlike other things like the framework, users mutating it won't mess with internals.